buildpacks/npm/shpec/build_shpec.sh

```diff
@@ -64,14 +64,6 @@ describe "lib/build.sh"
 
 		export CNB_STACK_ID="heroku-20"
 
-		it "creates store.toml when not present"
-			assert file_absent "$layers_dir/store.toml"
-
-			clear_cache_on_stack_change "$layers_dir"
-
-			assert file_present "$layers_dir/store.toml"
-		end
-
```

Comment:
We will still want to test the `write_to_store_toml` function on it's own in another unit test